### ISSUE TITLE
Pretrained weights for diffusion policy

### ISSUE THREAD

**Issue by siddancha** [ARISE-Initiative/robomimic#291]
Diffusion policy was added in the most recent version of robomimic (v0.5). Are pretrained weights available for the diffusion policy baseline? It is not present in the [model zoo](https://robomimic.github.io/docs/model_zoo/robomimic_v0.1.html).

**Comment by vaibhavsaxena11** [comment]
Thanks for raising this issue!

You’re right - pretrained weights for the Diffusion Policy (DP) baseline are not currently available in the robomimic model zoo. We will do our best to release DP model checkpoints within the next ~3 weeks.

In the meantime, you can train your own Diffusion Policy models by following the instructions in [this documentation](https://robomimic.github.io/docs/tutorials/training_diffusion_policy.html).

**Comment by siddancha** [comment]
Hi @vaibhavsaxena11, any updates on this?

Relatedly, we (@Geeksongs) trained the diffusion policy baseline locally for 50 epochs on Lift Cube task, and are seeing a success rate of 68%. Does that seem right?

**Comment by vaibhavsaxena11** [comment]
Hi @siddancha, Diffusion Policy (image-based) checkpoints for 5 tasks are now available on our [HF repo](https://huggingface.co/robomimic/robomimic_models/tree/main). Please find the summary of results, and links to download checkpoints below.

| **Task**           | **Success Rate** | **Checkpoint link** |
|--------------------|------------------|---------------------|
| Lift (PH)          | ~100%            | [[download](https://huggingface.co/robomimic/robomimic_models/blob/main/diffusion_policy_trained_models/lift-dp_ddpm/20260204091014/models/model_epoch_50_image_v15_success_1.0.pth)]          |
| Can (PH)           | ~100%            | [[download](https://huggingface.co/robomimic/robomimic_models/blob/main/diffusion_policy_trained_models/can-dp_ddpm/20260205023205/models/model_epoch_100_image_v15_success_1.0.pth)]          |
| Square (PH)        | ~90%             | [[download](https://huggingface.co/robomimic/robomimic_models/blob/main/diffusion_policy_trained_models/square-dp_ddpm/20260205023205/models/model_epoch_100_image_v15_success_0.9.pth)]          |
| Transport (PH)     | ~96%             | [[download](https://huggingface.co/robomimic/robomimic_models/blob/main/diffusion_policy_trained_models/transport-dp_ddpm/20260205033504/models/model_epoch_400_image_v15_success_0.96.pth)]          |
| Tool hang (PH)     | ~2%              | [[download](https://huggingface.co/robomimic/robomimic_models/blob/main/diffusion_policy_trained_models/tool_hang-dp_ddpm/20260205033504/models/model_epoch_100_image_v15_success_0.02.pth)]          |

Hope this is helpful!

**Comment by siddancha** [comment]
Thank you for sharing the checkpoints @vaibhavsaxena11. We tried to reproduce them (by @Geeksongs)  and here are our results:

<img width="600" alt="Image" src="https://github.com/user-attachments/assets/05892718-b86e-44fe-a6b0-9dff3d39ddb8" />

<br>
We're especially concerned about the "Transport (PH)" checkpoint, since there's a huge difference in the reproduction. Could you please confirm that the provided Transport (PH) checkpoint indeed produces 96% accuracy? Thanks so much!
